ntc_templates/templates/cisco_ios_show_spanning-tree_root.textfsm

```diff
@@ -0,0 +1,18 @@
+Value VLAN_ID (\S+)
+Value ROOT_ID ([0-9]{5}\s[0-9a-fA-F]{4}\.[0-9a-fA-F]{4}\.[0-9a-fA-F]{4})
```

Comment:
* Make STP priority more flexible with `\d+`
* Make the whitespace flexible by going from `\s` to `\s+`
* Simplify the MAC address regex

```suggestion
Value ROOT_ID (\d+\s+\S+)
```

Reply:
Opening this up to @jvanderaa @jmcgill298 @itdependsnetworks for thoughts/review. Thank you.

Reply:
should this be 2 separate values? like `ROOT_ID_INT` and `ROOT_ID_MAC`?

Reply:
> should this be 2 separate values? like `ROOT_ID_INT` and `ROOT_ID_MAC`?

Also a worthwhile point.

Maybe instead of `ROOT_ID` go with `PRIORITY` (the STP priority) and `MAC_ADDRESS`?